delegatednews/delegatednews-war/src/main/webapp/delegatednews/jsp/listNews.jsp

```diff
@@ -286,7 +286,7 @@
       <form name="tabForm" method="post">
   <%
     ArrayPane arrayPane = gef.getArrayPane("newsList", "Main", request, session);
-    arrayPane.setVisibleLineNumber(1000);
+    arrayPane.setVisibleLineNumber(GraphicElementFactory.getSettings().getInteger("Pagination.NumberPerPageThreshold",20));
```

Comment:
Why don't use the occasion to replace the Java code of ArrayPane by its counterpart `view:arrayPane`?

Reply:
I updated the ArrayPaneTag code so that the visible line number is initialized by default with the `Pagination.NumberPerPageThreshold` property. So setting explicitly the visible line number isn't anymore required for array pane.

Reply:
ok, i'm working on the replacement of the servlet code by jstl..

Reply:
done...